src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  flex-shrink: 0;
+  width: $card-width * 4 + $card-margin-right * 3;
```

Comment:
Ok, also you could set max-width and min-width. It's good that you used variable here